src/main/common/security/serviceTokenFactory.ts

```diff
@@ -0,0 +1,5 @@
+import ServiceAuthToken from 'app/idam/serviceAuthToken'
+
+export interface ServiceAuthTokenFactory {
```

Comment:
This should probably be in `service-2-service-api` module, can leave as a next step though...

Reply:
ok